packages/tasks/src/model-libraries.ts

```diff
@@ -394,6 +394,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/Tencent/HunyuanDiT",
 		countDownloads: `path:"pytorch_model_ema.pt" OR path:"pytorch_model_distill.pt"`,
 	},
+	"hunyuan3d-2": {
+		prettyLabel: "Hunyuan3D-2",
+		repoName: "Hunyuan3D-2",
+		repoUrl: "https://github.com/Tencent/Hunyuan3D-2",
+		countDownloads: `path_extension:"safetensors" OR path_extension:"ckpt"`,
```

Comment:
```suggestion
		countDownloads: `path:"model_index.json" OR path:"config.yaml"`,
```

Chatted with @Wauplin and we think that this is a good compromise as only one or the other are used and this is still quite future proof even when you split to other repos as well! Let me know if this doesn't work for you!

Reply:
Sure, this solution would be great! I just made an edition according to the suggesiton.